Libraries/LibJS/Interpreter.h

```diff
@@ -83,6 +83,21 @@ class Interpreter : public Weakable<Interpreter> {
         return interpreter;
     }
 
+    template<typename... Args>
+    [[nodiscard]] Value invoke(Function& function, Value this_value, Args... args)
```

Comment:
Can we maybe find a different name here or make this an overload of `call()` or something, I see confusion with `Object::invoke()` upon us - "invoke" is a specific thing in the spec: https://tc39.es/ecma262/#sec-invoke

https://github.com/SerenityOS/serenity/blob/bbe2d4a2d97ce7968b0985e4f99cdfc01fe5823e/Libraries/LibJS/Runtime/Object.cpp#L857-L868

Reply:
Hmmm...I got this name from `std::invoke`, but we can for sure name it `call()`